index.html

```diff
@@ -54,20 +54,124 @@
 
 
     <div class="page">
-        <div class="column" id="pocetna"></div>
+        <div class="column" id="home"></div>
     </div>
 
 
     <div class="page">
+
         <div class="column" id="second">
             <p>"Preventivna poreska kontrola vašeg poslovanja"</p>
+
+            <div class="service-row">
+
+                <div>
+                    <img alt="loading.." src="slike/slikaprva.png">
```

Comment:
svaka slika treba da ima jedinstvenu vrednost u alt atributu, to je kao placeholder koji se prikaze ako se slika ne ucita